amplfi/train/architectures/embeddings/multimodal.py

```diff
@@ -0,0 +1,67 @@
+from typing import Literal, Optional
+
+import torch
+from ml4gw.nn.norm import NormLayer
+from ml4gw.nn.resnet.resnet_1d import ResNet1D
+
+from amplfi.architectures.embeddings.base import Embedding
```

Comment:
```suggestion
from .base import Embedding
```

Reply:
@ravioli1369 since this is single repo now, could you also do relative imports in other places. Also, if you can add a unittest along the lines of the other ones.

Reply:
added, although it would need to be updated when we switch to injecting the frequency data directly into the background instead of ffting